src/main/java/uk/gov/ons/validation/entity/ValidationConfig.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.ons.validation.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ValidationConfig {
+    public String questionCode;
```

Comment:
There attributes should really be marked private.

Reply:
Done Leon